packages/react-docs/patternfly-a11y.config.js

```diff
@@ -91,7 +93,10 @@ module.exports = {
     'color-contrast',
     'landmark-no-duplicate-main',
     'landmark-main-is-top-level',
-    'scrollable-region-focusable'
+    'scrollable-region-focusable',
+    'link-in-text-block',
+    // temporarily enable this rule that has started failing test in v6 branch.
+    'aria-required-children'
```

Comment:
Just an FYI for others, https://github.com/patternfly/patternfly-react/issues/9968 should go towards removing this from this ignore array.